SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/HexahedronSetTopologyContainer.h

```diff
@@ -49,6 +49,9 @@ namespace sofa::component::topology
 {
 class HexahedronSetTopologyModifier;
 
+extern const unsigned int edgesInHexahedronArray[12][2];
+extern const unsigned int quadsInHexahedronArray[6][4];
+extern const unsigned int verticesInHexahedronArray[2][2][2];
```

Comment:
not sure to get the last one... 2 x 2 x 2, indead 8 vertices in an Hexa but what is the order?

Reply:
I am afraid I cannot help you for that...

Reply:
is it used?

Reply:
It is used in a few places, but from what I can see, those functions are not used. Check `HexahedronSetGeometryAlgorithms::isHexahedronAffine` and `HexahedronSetTopologyContainer::getLocalIndexFromBinaryIndex`. If we can remove them, I think we can remove this variable. What do you think?

Reply:
It is used in a few places, but from what I can see, those functions are not used. Check `HexahedronSetGeometryAlgorithms::isHexahedronAffine` and `HexahedronSetTopologyContainer::getLocalIndexFromBinaryIndex`. If we can remove them, I think we can remove this variable. What do you think?